base.launch.py

```diff
@@ -0,0 +1,195 @@
+import os
```

Comment:
What changes were made to this file? Is this not just a copy of the base.launch.py file? trailbot_demo1.launch should just be added as a launch in here